scripts/quests/crystalWar/WOTG_BAS_0_The_Fighting_Fourth.lua

```diff
@@ -0,0 +1,267 @@
+            {
+                [140] = function(player, csid, option, npc)
+                    if option == 1 then
+                        removeRations(player)
+                        returnLetter(player)
+                        quitQuest(player)
+                    end
+                end,
+
+                [141] = function(player, csid, option, npc)
+                    if option == 1 then
+                        returnLetter(player)
+                        quitQuest(player)
+                    end
+                end,
+
+                [142] = function(player, csid, option, npc)
+                    if option == 1 then
+                        returnLetter(player)
+                        quitQuest(player)
+                    end
+                end,
+
+                [143] = function(player, csid, option, npc)
+                    quest:complete(player)
+                end,
+            },
+        },
+
+        [xi.zone.NORTH_GUSTABERG_S] =
+        {
+            ['Gebhardt'] =
+            {
+                onTrigger = function(player, npc)
+                    local prog = quest:getVar(player, 'Prog')
+
+                    if prog == 0 then
+                        -- Rations!? Hand those over, rookie!
+                        return quest:progressEvent(102)
+                    elseif prog == 1 then
+                        return quest:progressEvent(103) -- Reminder
+                    elseif prog == 2 then
+                        -- Those turtlebacks intercepted the communication!?
+                        return quest:progressEvent(107)
+                    elseif prog == 3 then
+                        -- Thanks to your diligence, the Republic is safe for another day.
+                        return quest:progressEvent(108)
+                    end
+                end,
+            },
+
+            ['Roderich'] =
+            {
+                onTrigger = function(player, npc)
+                    local prog = quest:getVar(player, 'Prog')
+
+                    if prog == 1 then
+                        -- Centurion Gebhardt asked you to come all the way here...?
+                        return quest:progressEvent(104)
+                    elseif prog == 2 then
+                        return quest:progressEvent(105) -- Reminder
+                    elseif prog == 3 then
+                        -- I just heard the report!
+                        return quest:progressEvent(109)
+                    end
+                end,
+            },
+
+            ['Barricade'] =
+            {
+                onTrigger = function(player, npc)
+                    if quest:getVar(player, 'Prog') == 2 then
+                        -- Stop right there! I think you have something of mine!
+                        return quest:progressEvent(106)
+                    end
+                end,
+            },
+
+            onEventFinish =
+            {
+                [102] = function(player, csid, option, npc)
```

Comment:
these 3 cs are the only ones that should be progressEvent() and the others above should be event() as just filler dialog that are unblocking